### DISCUSSION TITLE
Incude Local Display Backlight In Power-Off

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1466] [Feature Requests]
@rern
I am a bit reluctant to request a new feature with so much going on, but would it be possible to include the command 
`echo 1 > /sys/class/backlight/rpi_backlight/bl_power` in the power-off sequence? At the moment the local display backlight stays on after power down.

**Comment by ceejayemm** [comment]
@R101G 

Just thinking out loud :-)

From system readme.md:

**Custom startup / shutdown script
Copy custom script named startup.sh / shutdown.sh to BOOT**

Why not try creating your own custom shutdown script which just includes the command you want ?

Chris

**Reply by R101G** [reply]
@ceejayemm 
Thanks for the suggestion. I will give it a try.

**Reply by R101G** [reply]
@ceejayemm 
My first bash script and it works a treat.

**Reply by ceejayemm** [reply]
@R101G 
Glad to be of service, now I know it works I will add the same too.

Chris

**Comment by klassikmann** [comment]
@ceejayemm @R101G 
Look at the UPDATE branch: https://github.com/rern/rAudio/commit/9dd010d54d03d1938897e0ff407f52cbb082538b

**Reply by R101G** [reply]
@klassikmann 
That doesn't work for me.

**Reply by Vseason** [reply]
Has there been any progress on this in UPDATE, does turning off the backlight now work, and will it migrate to main? I’d made a feature request about this the other day without realizing you already had a thread on the same subject. (Oops!)

**Reply by R101G** [reply]
@Vseason  I don't know what happened to this. Putting the command in shutdown.sh worked, and I never removed it. I have not seen anything from @rern about it.

**Reply by rern** [reply]
@R101G,

Please try:
```sh
sed -i '/alsactl/ a\
echo 1 > /sys/class/backlight/rpi_backlight/bl_power
' /srv/http/bash/power.sh
```

**Reply by R101G** [reply]
@rern Yes, thanks, that seems to work.

**Reply by Vseason** [reply]
@rern I’ve just tried this on a clean install of 20240108 (turned off the browser, used sed to input the suggested modification, turned the browser back on, rebooted the system, set a timeout, and waited), but so far the backlight is still on even when the screen goes off. Is there any additional step I should include for it to take effect?
